### PR TITLE
Minor edits - punctuation, spelling to make the contributing instructions more readable for all.

### PR DESCRIPTION
**Pull request by TysonsTech**
Greetings - Wanted to ease into my first Bitcoin core contribution with some text updates. Small changes but I think the doc reads a bit better. Cheers!

### CODE REVIEW

**Review by stickies-v — COMMENTED** [review]
NACK, most of these changes are subjective (e.g. American vs British spelling) and don't offer clear improvements, some are wrong (e.g. [this](https://github.com/bitcoin/bitcoin/pull/27082/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R237-R238))

Also, please see the contributor notes regarding [squashing commits](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits).

Thank you for contributing to the documentation! I'd prefer to be a bit more selective and only change things that are unequivocally wrong or offer substantial improvement. Unfortunately, in an open source community, review time is scarce.

**Comment by MarcoFalke** [comment]
Closing for now due to controversy.

In the future, please make sure to read the contribution guideline and make sure the tests pass (including the lint CI task).

**Comment by MarcoFalke** [comment]
## Getting started to contribute to Bitcoin Core

### Setting up your development environment

New developers are very welcome and needed. There are a lot of open issues of any difficulty waiting to be fixed. However, before you start contributing, familiarize yourself with the Bitcoin Core build system and tests. Refer to the documentation in the repository on how to build Bitcoin Core and how to run the unit and functional tests. Once that is done, you are all set.

If you need more help getting started, please refer to the following resources:

* https://medium.com/@amitiu/onboarding-to-bitcoin-core-7c1a83b20365
* https://jonatack.github.io/articles/how-to-review-pull-requests-in-bitcoin-core

### Pick something to work on

If you are looking for useful contributions to help out with, you can

* Search through the [good first issue](https://github.com/bitcoin/bitcoin/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)s or the ones that are [up for grabs](https://github.com/bitcoin/bitcoin/issues?utf8=%E2%9C%93&q=label%3A%22Up+for+grabs%22). Some of them might no longer be applicable. So if you are interested, but unsure, you might want to leave a comment on the issue first.
* Write tests to improve the coverage. Any kind of test is welcome and coverage information can be obtained from a relatively recent [coverage report](https://marcofalke.github.io/btc_cov). If you are unsure, don't hesitate to check back first.
* Help with review and testing. There are easy ones such as the [gui](https://github.com/bitcoin-core/gui/pulls) and [rpc](https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+is%3Aopen+label%3ARPC%2FREST%2FZMQ). However, review on any open pull request is welcome. Review will also help you understand the codebase better.
* Help on meta projects related to Bitcoin Core, such as a [high-level performance monitor](https://github.com/chaincodelabs/bitcoinperf).
* Join us on irc and let us know what you are interested in.

**Review by jonatack — COMMENTED** [review]
Agree that at least several of the changes are incorrect, and the other improvements probably aren't worth the review time being asked for. 

Generally, spelling and grammar edits are usually not a good way to begin contributing to this particular project. One of the best ways would be to start with reviewing and testing. See https://jonatack.github.io/articles for more.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -272,18 +271,18 @@ There are three categories of refactoring: code-only moves, code style fixes, an
...
 
 Pull requests that refactor the code should not be made by new contributors. It
 requires a certain level of experience to know where the code belongs to and to
 understand the full ramification (including rebase effort of open pull requests).
 
 Trivial pull requests or pull requests that refactor the code with no clear
-benefits may be immediately closed by the maintainers to reduce unnecessary
-workload on reviewing.
+benefits may be immediately closed by the maintainers to reduce unnecessarily
+the workload on reviewing.
```
[the remark this answers is not in the document]
Comment (TysonsTech) [reply CONTRIBUTING.md:285]: Appreciate the feedback, and the helpful article links! Thank you, sir!

**Pull request closed without merging**